src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Models/TestDto.cs

```diff
@@ -26,19 +37,26 @@ public class TestDto
 
         public string Discount { get; set; }
 
-        public string DiscountUrl { get; set; }
+        public Uri DiscountUrl { get; set; }
 
-        public string DiscountLogoUrl { get; set; }
+        public Uri DiscountLogoUrl { get; set; }
 
         public string DiscountLogoPath { get; set; }
 
         public bool HasUserAnswered { get; set; }
 
         public bool IsAdvent => DateTime.Now.Month == 12 && DateTime.Now.Day < 25;
-    }
 
-    public enum TestStatus
-    {
-        NotStarted, Started, Ended
+        public static implicit operator TestDto(TestDto v)
+        {
+#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
+            throw new NotImplementedException();
+#pragma warning restore CA1065 // Do not raise exceptions in unexpected locations
+        }
+
+        public TestDto ToTestDto()
+        {
+            throw new NotImplementedException();
```

Comment:
please add //TODO comment to fix it - or remove it, if it is not used

Reply:
Deleted.